bolt/lib/Core/BinaryContext.cpp

```diff
@@ -1945,6 +1959,29 @@ static void printDebugInfo(raw_ostream &OS, const MCInst &Instruction,
     OS << " discriminator:" << Row.Discriminator;
 }
 
+/// Skip instructions that do not potentially manipulate or compare function
+/// addresses.
+static bool skipInstruction(const MCInst &Inst, const BinaryContext &BC) {
+  const bool IsX86 = BC.isX86();
```

Comment:
What is the purpose of defining `IsX86` here?

Reply:
I think I just miss understood the code I was looking at for references.